### ISSUE TITLE
[ERROR]	Cannot specify both gene file and set file

### ISSUE THREAD

**Issue by weizhousjtu** [zhanxw/rvtests#88]
Hi, everyone. I have an error "[ERROR]	Cannot specify both gene file and set file"  when doing setbased analysis. my set file is :
` phosphatases	chr3:190282078-190347307`
my geneFile is refFlat_hg19.txt.gz
when I try the example set file , I have the same error. 
#### Any one could help me? Thank you very much!
example set file:
`set1 1:100-200,1:250-300
set2 2:500-600`

**Comment by zhanxw** [comment]
You're right. Can you perform one analysis using gene files and another using the set files?
